.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,44 @@
-**Description:**
+## First
+- [ ] I have read and understand the [Contributing](https://github.com/home-assistant/home-assistant/CONTRIBUTING.md) documentation.
 
+## Description
+<!--- Describe your changes. -->
 
-**Related issue (if applicable):** fixes #
+## Types of Changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
-**Pull request in [home-assistant.io](https://github.com/home-assistant/home-assistant.io) with documentation (if applicable):** home-assistant/home-assistant.io#
+## Issues
+<!-- List all issues that your change addresses ex( - home-assistant/home-assistant/issues/3745 ) -->
 
-**Example entry for `configuration.yaml` (if applicable):**
-```yaml
+## Pull Requests
```

Comment:
Again, let's focus on keeping this simple. We used to call out the link to documentation because we wanted people to focus on docs. 

There is no need to point at the polymer/javascript PRs. Anyone that has a related one has linked that one in the past.